eng/Packages.Data.props

```diff
@@ -354,7 +354,7 @@
     <PackageReference Update="Polly.Contrib.WaitAndRetry" Version="1.1.1" />
     <PackageReference Update="Portable.BouncyCastle" Version="1.9.0" />
     <PackageReference Update="PublicApiGenerator" Version="10.0.1" />
-    <PackageReference Update="System.ClientModel" Version="1.2.0" />
+    <PackageReference Update="System.ClientModel" Version="1.2.1" />
```

Comment:
This change already exists in main and should not appear in this PR.

Reply:
Ok, I was getting build errors. I will pull the changes from main.

Reply:
I have merged from the source branch, so it should resolve as no change, right?

Reply:
Rebase on main, don't merge.   You want to put your changes on top of the latest.   There should be no files in this PR that you're not changing for your work.

Reply:
rebased to https://github.com/Azure/azure-sdk-for-net.git